apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/StylingCard.tsx

```diff
@@ -0,0 +1,209 @@
+"use client";
+
+import { Placement } from "@/app/(app)/environments/[environmentId]/settings/lookandfeel/editLookAndFeel";
+import { PlacementType } from "@formbricks/types/js";
+import type { Survey } from "@formbricks/types/surveys";
+import { ColorPicker, Label, Switch } from "@formbricks/ui";
+import { CheckCircleIcon } from "@heroicons/react/24/solid";
+import * as Collapsible from "@radix-ui/react-collapsible";
+import { useState } from "react";
+
+interface StylingCardProps {
+  localSurvey: Survey;
+  setLocalSurvey: (survey: Survey) => void;
+}
+
+export default function StylingCard({ localSurvey, setLocalSurvey }: StylingCardProps) {
+  const [open, setOpen] = useState(false);
+
+  const {
+    brandColor,
+    placement,
+    darkOverlay,
+    clickOutsideClose,
+    type,
+    highlightBorderColor,
+    overwriteBorderHighlight,
+  } = localSurvey;
+
+  const isBrandColor = brandColor !== null;
+  const ishighlightBorder = highlightBorderColor !== null;
+  const isPosition = placement !== null;
+
+  const togglePlacement = () => {
+    if (isPosition) {
+      const updatedSurvey: Survey = {
+        ...localSurvey,
+        placement: null,
+        clickOutsideClose: true,
+        darkOverlay: false,
+      };
+      setLocalSurvey(updatedSurvey);
+    } else {
+      const updatedSurvey: Survey = { ...localSurvey, placement: "bottomRight" };
+      setLocalSurvey(updatedSurvey);
+    }
+  };
+
+  const toggleBrandColor = () => {
+    if (isBrandColor) {
+      setLocalSurvey({ ...localSurvey, brandColor: null });
+    } else {
+      setLocalSurvey({ ...localSurvey, brandColor: "#64748b" });
+    }
+  };
+
+  const toggleBorderColor = () => {
+    if (ishighlightBorder) {
+      setLocalSurvey({ ...localSurvey, highlightBorderColor: null });
+    } else {
+      setLocalSurvey({ ...localSurvey, highlightBorderColor: "#64748b" });
+    }
+  };
+
+  const toggleBorderColorSetting = () => {
```

Comment:
please remove this and only use the current value of `localSurvey.highlightBorderColor` to determine the current state (null => deactivated) 😊